index.html

```diff
@@ -136,31 +204,59 @@ <h4>Don't Forget These!</h4>
         </ul>
       </div>
     </div>
-    <div class="row">
+    <div class="row mt-4">
       <div class="col-sm-12">
         <form>
-          <label for="name">Name</label>
-          <input id="name" type="text" class="form-control" placeholder="Name">
-          <label for="phone">Phone</label>
-          <input id="phone" type="phone" class="form-control" placeholder="Phone">
-          <div class="form-check">
-            <input class="form-check-input" type="radio" name="exampleRadios" id="exampleRadios1" value="option1" checked>
-            <label class="form-check-label" for="exampleRadios1">
-              Beach
-            </label>
+            <div class="form-group">
+              <label for="exampleInputEmail1">Email address</label>
+              <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
```

Comment:
In case it escaped your attention: By using `input type="email"` you're adding some nice built-in validation for that field with Bootstrap :smile: